packages/react-storage/src/components/StorageBrowser/controls/hooks/__tests__/useActionStart.spec.ts

```diff
@@ -0,0 +1,43 @@
+import { useControlsContext } from '../../../controls/context';
+import { ControlsContext } from '../../types';
+import { useActionStart } from '../useActionStart';
+
+jest.mock('../../../controls/context');
+
+describe('useActionStart', () => {
+  const controlsContext: ControlsContext = {
+    data: {
+      taskCounts: {
+        CANCELED: 2,
+        COMPLETE: 4,
+        FAILED: 3,
+        INITIAL: 0,
+        PENDING: 0,
+        QUEUED: 1,
+        TOTAL: 10,
```

Comment:
Nit: Probably not necessary to have these in the mock as they're just ignored anyway